docs/package.json

```diff
@@ -1,33 +1,37 @@
 {
   "name": "remeda-docs",
   "version": "1.0.0",
-  "main": "index.js",
   "repository": "git@github.com:remeda/docs.git",
   "author": "Łukasz Sentkiewicz",
   "license": "MIT",
   "scripts": {
-    "start": "node index",
-    "watch": "parcel serve index.html",
-    "build": "cross-env NODE_ENV=production parcel build index.html --public-url ./  --out-file index.html && cp _redirects dist/_redirects && cp dist/index.html dist/404.html"
+    "watch": "vite",
+    "build": "tsc && vite build",
```

Comment:
one more thing, you dropped the copying of the `_redirects` file to the dist folder, but I think that would cause the site to break if someone tries to access a deep link into the docs directly (and not by going to `/` and browsing from there). Can you validate that this isn't broken right now...?

Reply:
I will check gh-pages document!

Reply:
@eranhirsch 

I'm sorry, I misunderstood, but is https://remedajs.com/ deployed with Netlify?

I looked at .github/workflows/pages.yml and thought it was deployed on github pages, but looking at the response header on remedajs.com, I can see that it came from netlify.

![image](https://github.com/remeda/remeda/assets/52757302/e5144c8a-5c29-407c-8970-f7bf80145eca)

Reply:
https://github.com/remeda/remeda/pull/359/commits/5d990066c590583963e2f82b5f0fa97fa9e23367

For the time being, I inserted the _redirects file inside the dist folder.